### DISCUSSION TITLE
[FeatureRequest] Browse controls from a now time range

### DISCUSSION THREAD

**Discussion by christianTF** [grafana/grafana#38263] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Currently, if a relative time range is set (now-7d), the browse arrows < > are not shown.

Only when I set absolute time ranges, these buttons are shown.

I would like that the < is also shown with a relative time range.
If I press the < button, Grafana should automatically switch to absolute time range, calculating the absolute start and end based on the set relative range.

**Why is this needed**:
It is more intuitive to start with a pre-defined relative time range to browse back in time (starting to scroll back from now).

Currently, it is quite complicated to start from now and then change and enter absolute time ranges.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
